packages/suite/src/support/messages.ts

```diff
@@ -3072,9 +3093,21 @@ export default defineMessages({
         id: 'TR_YOU_WERE_DISCONNECTED_DOT',
     },
     TR_YOUR_CURRENT_FIRMWARE: {
-        defaultMessage: 'Your current firmware version is {version}',
+        defaultMessage: 'Your current version is {version}',
         id: 'TR_YOUR_CURRENT_FIRMWARE',
     },
+    TR_YOUR_CURRENT_FIRMWARE_TYPE: {
+        defaultMessage: 'Your firmware type is {version}',
+        id: 'TR_YOUR_CURRENT_FIRMWARE_TYPE',
+    },
+    TR_SWITCH_TO_BITCOIN: {
+        defaultMessage: 'Switch to Bitcoin',
```

Comment:
Switch to Bitcoin-only?

Reply:
This would make the button longer than others so I would keep this as an exception to the terminology.